src/Athena/Content/js/studentSetup.js

```diff
@@ -41,10 +41,15 @@ function makeCard(id, title, description) {
                     <div class="col xl4 s6">
                         <div class="card grey darken-2 white-text">
                             <div class="card-content">
-                                <span class="card-title activator"></span>
+                                <span class="card-title activator"> 
+                                    <span class="card-title-target"> </span>
+                                     <i class="material-icons right">info</i>
+                                </span>
                             </div>
                             <div class="card-reveal grey darken-2 white-text">
-                                <span class="card-title"></span>
+                               <span class="card-title activator"> 
```

Comment:
I think we should add a close icon here to hint that it can be closed. Thoughts?

![peek 2018-04-17 13-11](https://user-images.githubusercontent.com/794251/38885475-0d1a1df8-4241-11e8-87cd-8a201b5a26b6.gif)